### PR TITLE
Update client logging

### PR DESCRIPTION
**Pull request by davidfischer**
- Exposes control of logging with `data-ea-verbosity`
- Makes some debug/info messages not visible by default
- Downgrades "already loaded" from error->warning
- Increases "No ad placements found" (missing div) from debug->warning

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -921,7 +956,7 @@ export function load_placements(force_load = false) {
   let elements = Array.prototype.slice.call(node_list);
 
   if (elements.length === 0) {
-    console.debug("No ad placements found.");
+    logger.warn("No ad placements found.");
```
Comment (ericholscher) [inline index.js:959]: Should we provide more context here? I feel like as a user this message is not super helpful..

```suggestion
    logger.warn("No ad placements found. Check your publisher account is correct and has ads enabled.");
```

File: index.js [hunk 2]
```diff
@@ -36,6 +36,41 @@ const AD_TYPES_VERSION = 1; // Used with the ad type slugs
...
+const VERBOSITY = {
+  quiet: 0, // Errors only
+  normal: 1, // Warnings only (default)
+  verbose: 2, // Debug messages
+};
+const logger = {
+  verbosity: VERBOSITY["normal"], // Default
+
+  debug(message, ...params) {
+    if (this.verbosity >= 2) {
```
Comment (ericholscher) [inline index.js:53]: These should use the VERBOSITY names, and not ints, right? Seems more explicit.

Comment (ericholscher) [reply index.js:53]:
```suggestion
    if (this.verbosity >= VERBOSITY["verbose"]) {
```

**Review by ericholscher — APPROVED** [review]

File: index.js (hunk 2 above)
Comment (davidfischer) [reply index.js]: Done

File: index.js [hunk 3]
```diff
@@ -1071,17 +1122,17 @@ if (check_dependencies()) {
           if (err instanceof Error) {
             if (err instanceof EthicalAdsWarning) {
               // Report these at a lower log level
-              console.debug(err.message);
+              logger.warn(err.message);
```
Comment (davidfischer) [inline index.js:1125]: I think we want these at warning. The main things that are displayed from this message are when there are no ads to show and when the ad request is blocked (ad blocker) or invalid (wrong publisher account). I think we want these to be visible at default verbosity.

Comment (ericholscher) [reply index.js:1127]: Yea, makes sense 👍

**Pull request merged**